Kittygram.xcodeproj/project.pbxproj

```diff
@@ -627,10 +633,10 @@
 			baseConfigurationReference = E12AD690597F9DCDD35297AC /* Pods-Kittygram.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = CCHRC6DB43;
+				DEVELOPMENT_TEAM = P75L6RL67A;
 				INFOPLIST_FILE = Kittygram/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.sunshinejr.Kittygram;
```

Comment:
Heh, I know you probably couldn't build it for the device due to my certs & bundle id in the project, but could we remove this certs changes?